### DISCUSSION TITLE
Carousel slide animations

### DISCUSSION THREAD

**Discussion by adiluhungsetya** [themesberg/flowbite-react#427] [Q&A]
I am using flowbite-react base theme, and after i use it there is no animation effect in slider, must i use custom theme to enable the animation ?

**Comment by rluders** [comment]
That is actually a good question. I probably would say that by now you could probably just use `className` at the component, but remember that you need to `!` make it important to overwrite the default theme.

**Comment by tulup-conner** [comment]
It probably wouldn't be a bad idea for us to add a default animation and some animation options.
